Formula/gtk4.rb

```diff
@@ -1,4 +1,4 @@
-class Gtkx4 < Formula
+class Gtk4 < Formula
```

Comment:
Please add an alias for the original name, people are still going to try and use it.